Cubical/Functions/Embedding.agda

```diff
@@ -425,3 +425,17 @@ module EmbeddingIdentityPrinciple {B : Type ℓ} {ℓ₁} (f g : Embedding B ℓ
       f ≡ g
     ■
 open EmbeddingIdentityPrinciple renaming (f≃g to _≃Emb_) using (EmbeddingIP) public
```

Comment:
Why open public?

Reply:
As far as I know, this is not code I wrote. Good question though.

Reply:
There is another instance of this in the file. Should I just change both?